packages/venia-concept/_buildpack/create.js

```diff
@@ -133,7 +133,14 @@ function setDebugDependencies(fs, pkg) {
     );
     const overridden = {};
     const workspaceDir = resolve(__dirname, '../../');
-    fs.readdirSync(workspaceDir).forEach(packageDir => {
+    const extensionDir = resolve(workspaceDir, 'extensions');
+
+    const packageDirs = [
+        ...fs.readdirSync(workspaceDir),
+        ...fs.readdirSync(extensionDir).map(path => `extensions/${path}`)
+    ];
+
```

Comment:
This works, but I think it would be a little more robust to just ask Yarn what the current packages are.

```js
    // Assume we are in the pwa-studio repo and get its root.
    const monorepoDir = resolve(__dirname, '../../../');

    // The Yarn "workspaces info" command outputs JSON as of v1.22.4.
    // The -s flag suppresses all other non-JSON logging output.
    const yarnWorkspaceInfoCmd = 'yarn -s workspaces info';
    let workspaceInfo = require('child_process').execSync(
        yarnWorkspaceInfoCmd,
        { cwd: monorepoDir }
    );

    let packageDirs;
    try {
        packageDirs = Object.values(JSON.parse(workspaceInfo)).map(
            ({ location }) => resolve(monorepoDir, location)
        );
    } catch (e) {
        // In case Yarn breaks this in the future, throw a more informative
        // error than your typical JSONError.
        throw new Error(
            `Could not parse output of '${yarnWorkspaceInfoCmd}:\n${workspaceInfo}`
        );
    }
```

Reply:
Love it, made this change in 9a0ce84.